crates/sui-rpc-api/proto/sui/rpc/v2alpha/move_descriptor.proto

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package sui.rpc.v2alpha;
+
+// Describes a data type.
+message Descriptor {
```

Comment:
Should this by `DatatypeDescriptor`? The fact that type descriptors are just `Descriptor` but function descriptors are qualified felt a bit strange when reading the module definition.